activestate.yaml

```diff
@@ -262,7 +264,13 @@ scripts:
       unset ACTIVESTATE_ACTIVATED
       export INTEGRATION_TEST_USERNAME=$secrets.project.INTEGRATION_TEST_USERNAME
       export INTEGRATION_TEST_PASSWORD=$secrets.project.INTEGRATION_TEST_PASSWORD
-      go test `go list ./... | grep "integration\|automation"` -v "${@:1}" -timeout 20m
+      go test `go list ./... | grep "${INTEGRATION_TEST_REGEX}"` -v "${@:1}" -timeout 20m
+  - name: integration-tests-build-only
+    language: bash
+    description: Builds integration tests and removes the .test artifacts.
+    value: |
+      go list ./... | grep "${INTEGRATION_TEST_REGEX}" | xargs --max-lines=1 go test -c
+      find . -type f -name "*.test" | grep "${INTEGRATION_TEST_REGEX}" | xargs rm
```

Comment:
What's the point of this? Doesn't seem related to the bug this PR is solving.

Reply:
By building the tests, incorrect/outdated usage of types within the tests will be caught by the compiler.